src/deno/build.ts

```diff
@@ -10,22 +10,19 @@ import {
   extname,
 } from "./deps.ts";
 import vendor from "../vendor.ts";
-import {
-  apiDirectory,
-  root,
-  sourceDirectory,
-  vendorDirectory,
-} from "../env.ts";
+import { apiDirectory, sourceDirectory, vendorDirectory } from "../env.ts";
 import { ImportMap } from "./../types.ts";
 
-const ultra = "https://deno.land/x/ultra";
+const root = Deno.env.get("root") || "http://localhost:8000";
```

Comment:
Does this need to be changed to use env? Thought root was deprecated for origin?

Reply:
Yes, I added that only in the build script -- as I think it still needs it, I have yet to confirm if we can remove it